recipes/pycudadecon/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "pycudadecon" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+
+  url: https://github.com/tlambert03/pycudadecon/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: da21f5a43ca6ebe171d7234fe0e3447bc2956e1d9ebe833da13da0fed00ae3f1
+
+build:
+  skip: True  # [osx]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+  host:
+    - python
+    - pip
```

```suggestion
    - pip
    - setuptools_scm
```

sorry ... forgot this one